app/javascript/components/widgets/weather/current-weather.test.jsx

```diff
@@ -10,13 +10,57 @@ describe('AnticipatedPrecipitation component', () => {
         precip.push({ precipitation: 0.5 });
       }
     }
-    const weather = { minutely: precip };
+    let byHour =
+      '[\n' +
+      '    {\n' +
+      '      "weather": {\n' +
+      '        "id": 804\n' +
+      '      }\n' +
+      '    },\n' +
+      '    {\n' +
+      '      "weather": {\n' +
+      '        "id": 804\n' +
+      '      }\n' +
+      '    }]';
+    byHour = JSON.parse(byHour);
```

Comment:
Maybe just start off as a json object instead of a string that gets parsed?

Reply:
Yeah that would've made a little more sense ;)
Changed.